zh/restore-from-azblob-using-br.md

```diff
@@ -46,105 +64,158 @@ summary: 介绍如何使用 BR 恢复 Azure Blob Storage 上的备份数据。
+本节假设 Azure Blob Storage 中的桶 `my-container` 中存在两份备份数据，分别是：
+
+- 在**日志备份期间**，进行快照备份产生的备份数据，存储在 `my-full-backup-folder-pitr` 文件夹下。
+- 日志备份产生的备份数据，存储在 `my-log-backup-folder-pitr` 文件夹下。
+
+关于如何备份数据，请参考[使用 BR 备份 TiDB 集群数据到 Azure Blob Storage](backup-to-azblob-using-br.md)。
+
+> **注意：**
+>
+> 指定的恢复时间点需要在快照备份时刻点之后，日志备份 `checkpoint-ts` 之前。
+
+### 第 1 步：准备恢复环境
+
+使用 BR 将 Azure Blob Storage 上的备份数据恢复到 TiDB 前，请按照以下步骤准备恢复环境。
+
+1. 创建一个用于管理恢复的 namespace，这里创建了名为 `restore-test` 的 namespace。
 
     ```shell
-    kubectl apply -f resotre-azblob.yaml
+    kubectl create namespace restore-test
     ```
 
-    `restore-azblob.yaml` 文件内容如下：
+2. 下载文件 [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml)，并执行以下命令在 `restore-test` 这个 namespace 中创建备份需要的 RBAC 相关资源：
 
-    ```yaml
-    ---
-    apiVersion: pingcap.com/v1alpha1
-    kind: Restore
-    metadata:
-      name: demo2-restore-azblob
-      namespace: test2
-    spec:
-      br:
-        cluster: demo2
-        clusterNamespace: test2
-        # logLevel: info
-        # statusAddr: ${status_addr}
-        # concurrency: 4
-        # rateLimit: 0
-        # timeAgo: ${time}
-        # checksum: true
-        # sendCredToTikv: true
-      # # Only needed for TiDB Operator < v1.1.10 or TiDB < v4.0.8
-      # to:
-      #   host: ${tidb_host}
-      #   port: ${tidb_port}
-      #   user: ${tidb_user}
-      #   secretName: restore-demo2-tidb-secret
-     azblob:
-        secretName: azblob-secret
-        container: my-container
-        prefix: my-folder
+    ```shell
+    kubectl apply -f backup-rbac.yaml -n restore-test
     ```
 
-+ 方法 2: 如果通过了 Azure AD 的方式授权，你可以按照以下说明创建 `Restore` CR 恢复集群数据：
+3. 为刚创建的 namespace `restore-test` 授予远程存储访问权限，可以使用两种方式授予权限，可参考文档 [Azure 账号授权](grant-permissions-to-remote-storage.md#azure-账号授权)。创建成功后, namespace `restore-test` 就拥有了名为 `azblob-secret` 或 `azblob-secret-ad` 的 secret 对象。
+
+    > **注意：**
+    >
+    > 授予的账户所拥有的角色至少拥有对 blob 访问的权限（例如[读取器](https://learn.microsoft.com/zh-cn/azure/role-based-access-control/built-in-roles#reader)）。
+    >
+    > 在创建 secret 对象的时候，你可以自定义它的名字。下文为了叙述简洁，统一使用名为 `azblob-secret` 的 secret 对象。
+
+### 第 2 步：将指定备份数据恢复到 TiDB 集群
 
-    {{< copyable "shell-regular" >}}
+本节示例中首先将快照备份恢复到集群中，因此 PITR 的恢复时刻点需要在[快照备份的时刻点](backup-to-azblob-using-br.md#查看快照备份的状态)之后，并在[日志备份的最新恢复点](backup-to-azblob-using-br.md#查看日志备份的状态)之前。具体步骤如下：
+
+1. 在 `restore-test` 这个 namespace 中产生一个名为 `demo3-restore-azblob` 的 `Restore` CR，并指定恢复到 `2022-10-10T17:21:00+08:00`:
 
     ```shell
-    kubectl apply -f restore-azblob.yaml
+    kubectl apply -f restore-point-azblob.yaml
     ```
 
-    `restore-azblob.yaml` 文件内容如下：
+    `restore-point-azblob.yaml` 文件内容如下：
 
     ```yaml
     ---
     apiVersion: pingcap.com/v1alpha1
     kind: Restore
     metadata:
-      name: demo2-restore-azblob
-      namespace: test2
+    name: demo3-restore-azblob
+    namespace: restore-test
     spec:
-      serviceAccount: tidb-backup-manager
+      restoreMode: pitr
```

Comment:
It looks we need some explanation for restoreMode?

Reply:
done